public/app/features/explore/QueryRows.tsx

```diff
@@ -116,19 +116,19 @@ class QueryRow extends PureComponent<QueryRowProps> {
 
 type QueryRowsProps = QueryRowCommonProps &
   QueryRowEventHandlers & {
-    queries: Query[];
+    initialQueries: DataQuery[];
```

Comment:
can you explain the naming around initialQueries here & for QueryRow, what is initial, is there a "later" query?